src/core/border_router/routing_manager.cpp

```diff
@@ -630,8 +628,13 @@ void RoutingManager::StartRoutingPolicyEvaluationDelay(void)
     static_assert(kMaxRoutingPolicyDelay > 0, "invalid maximum routing policy evaluation delay");
     randomDelay = Random::NonCrypto::GetUint32InRange(0, Time::SecToMsec(kMaxRoutingPolicyDelay));
 
-    otLogInfoBr("Start evaluating routing policy, scheduled in %u milliseconds", randomDelay);
-    mRoutingPolicyTimer.Start(randomDelay);
+    StartRoutingPolicyEvaluationDelay(randomDelay);
+}
+
+void RoutingManager::StartRoutingPolicyEvaluationDelay(uint32_t aDelayMilli)
```

Comment:
may combine with `StartRoutingPolicyEvaluationDelay()` with a special `aDelayMilli(0)` for randomdelay generation?

Reply:
Should we change `StartRoutingPolicyEvaluationDelay()` to `StartRoutingPolicyEvaluationDelayJitter()` ?

Reply:
my original proposal was to `combine the two StartRoutingPolicyEvaluationXXX()` into one method like `StartRoutingPolicyEvaluation(uint32_t aDelayMilli)` and generate the randomdelay when when aDelayMilli=0.

no strong preference if you like this way.

Reply:
`StartRoutingPolicyEvaluationDelay(0)` is actually used to schedule a Routing Policy evaluation right now.
Besides, seems we actually have two jitters:
- kRaReplyJitter for replying to RS
- kRoutingPolicyEvaluationJitter for routing policy re-evaluation